### PR TITLE
fix: pruning-at-boot default

### PR DESCRIPTION
**Pull request by gfukushima**

### CODE REVIEW

File: docs/how-to/configure-slashing-protection.md [hunk 1]
```diff
@@ -138,16 +138,16 @@ Enable pruning by setting
 [`--slashing-protection-pruning-enabled`](../reference/cli/subcommands.md#slashing-protection-pruning-enabled)
 to `true`.
 
-By default, Web3Signer prunes the database on startup, and then after each pruning interval.
-You can disable pruning at startup and start pruning after the pruning interval by setting
+Web3Signer can prune the database on startup, and then after each pruning interval.
+By default, this feature is disable. You can enable pruning at startup and start pruning after the pruning interval by setting
```
Comment (jframe) [inline docs/how-to/configure-slashing-protection.md:142]:
```suggestion
By default, this feature is disabled. You can enable pruning at startup and start pruning after the pruning interval by setting
```

**Review by siladu — CHANGES_REQUESTED** [review]
Needs changing here too https://docs.web3signer.consensys.net/reference/cli/subcommands#slashing-protection-pruning-at-boot-enabled

File: docs/reference/cli/subcommands.md [hunk 2]
```diff
@@ -1693,28 +1693,28 @@ The default is `true`.
...
+eth2.slashing-protection-pruning-at-boot-enabled: true
 ```
 
 <!--/tabs-->
 
-When set to `false`, [slashing protection database
+When set to `true`, [slashing protection database
 pruning](../../how-to/configure-slashing-protection.md#prune-the-slashing-protection-database) is
-disabled at boot and only takes place at the scheduled [pruning intervals](#slashing-protection-pruning-interval).
+enabled at boot as well as in the defined interval [pruning intervals](#slashing-protection-pruning-interval).
```
Comment (alexandratran) [inline docs/reference/cli/subcommands.md:1715]:
```suggestion
enabled at startup and at the defined [pruning intervals](#slashing-protection-pruning-interval).
```

**Review by alexandratran — APPROVED** [review]

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**